cmd/commands.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/ardikabs/shorty/kutt"
+	"github.com/spf13/cobra"
+)
+
+var api kutt.API
+
+func init() {
+	rootCmd.AddCommand(listCmd)
+	rootCmd.AddCommand(submitCmd)
+	rootCmd.AddCommand(deleteCmd)
+
+	apiToken := os.Getenv("KUTT_TOKEN")
+	if apiToken == "" {
+		fmt.Println("Kutt API Token are not set. 'KUTT_TOKEN'")
+		os.Exit(1)
+	}
+
+	api = kutt.API{
+		BaseURL: &url.URL{
+			Scheme: "https",
+			Host:   "kutt.it",
+		},
+		APIToken:     apiToken,
+		CustomDomain: os.Getenv("KUTT_CUSTOM_DOMAIN"),
+	}
+}
+
+var rootCmd = &cobra.Command{
+	Use:   "shorty",
```

Comment:
since this is named `shorty`, would be good to create a `Makefile` that includes how to build this code into name `shorty`, and how to test the whole repo.

```
Shorty CLI. Use with calm

Usage:
  shorty [command]

Available Commands:
  delete      Delete shorter URL
  help        Help about any command
  list        List shorter URL
  submit      Submit target URL to be shorten

Flags:
  -h, --help   help for shorty

Use "shorty [command] --help" for more information about a command.
```

This help page would confuse user as they might build `main.go` and binary output would be named `main`

Reply:
solved after add Makefile